graph/builder.go

```diff
@@ -394,28 +396,34 @@ func (b *Builder) syncGraphWithChain() error {
 			// can exit as this entails we are back to the point
 			// where it hasn't seen any block or created channels,
 			// alas there's nothing left to prune.
-			case err == channeldb.ErrGraphNeverPruned:
+			case errors.Is(err, channeldb.ErrGraphNeverPruned):
```

Comment:
Partially ignore my previous comment... Could still combine the outer `if` and the `switch` to remove one level of indentation?